tfhe/docs/tutorials/ascii_fhe_string.md

```diff
@@ -0,0 +1,148 @@
+# A first complete example: FheLatinString (Integer)
```

Comment:
Were there changes apart from latin -> ascii ?

Reply:
Yes, ASCII characters which are not letters are left unmodified
Hence the rename
But I was expecting the diff to detect a rename rather than treat this file as a new one